### DISCUSSION TITLE
New Sources: IHS Markit - iLEVEL Portfolio Monitoring Software

### DISCUSSION THREAD

**Discussion by xoChrisCo** [airbytehq/airbyte#33157] [New Connector Request]
## Tell us about the new connector you’d like to have
* Source: IHS iLevel Portfolio Monitoring Software (https://ihsmarkit.com/products/ilevel.html)
* Destination: BigQuery

## Describe the context around this new connector
* iLevel is used to collect (financial reporting) data from portfolio companies for private equity and other investment companies. FSN Capital Partners AS use it to collect monthly, quarterly, and annual reports from 20+ companies. Our portfolio companies report financial-, emission-, and KPI-data through iLevel that we wish to visualize in PowerBI or other analytical tools.
* iLevel can be connected to through an API (https://sand-docs.ilevelsolutions.eu/) With the enpoint in question: https://api.ilevelsolutions.eu/v1
* This is a paid API. You are free to name FSN Capital Partners AS as a mutual user for the account.

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Build a bespoke nodejs application that pulls the data into bigquery

## Are you willing to submit a PR?

Remove this with your answer :-)

**Comment by harshithmullapudi** [comment]
Hey @Chr1sCon will you be able to contribute this source?

**Comment by xoChrisCo** [comment]
Hey, sorry for the late reply @harshithmullapudi !

Absolutely!
I am a bit new to Airbyte, but am about to go through the [any% speedrun](https://docs.airbyte.com/connector-development/tutorials/cdk-speedrun) (love the naming, by the way), and build it myself. Happy to share as I get something working!

How does it work? If it passes some sort of quality check; can this be adopted as an official source? I would love to understand a bit more on the process for this!

[a comment by an automated account is not shown]
